src/nccl_ofi_topo.c

```diff
@@ -18,8 +18,13 @@
 #include "nccl_ofi_topo.h"
 #include "nccl_ofi_math.h"
 
+#if HAVE_CUDA
 static const uint8_t display_controller_class_id = 0x03;
 static const unsigned short nvidia_vendor_id = 0x10de;
+#else
```

Comment:
`#elif HAVE_NEURON`, for clarity if nothing else? Same for other similar `#else` clauses.